app/src/main/java/com/github/se/cyrcle/ui/theme/atoms/Button.kt

```diff
@@ -193,6 +192,36 @@ fun ToggleButton(
       }
 }
```

Comment:
Overload + modify an unused atom seems a bit overkill, maybe it would be simpler to tune the atom to your needs.

Reply:
Tried to. The two are not really compatible.

Reply:
I've looked it at multilple angles and this is what makes the most sense.

Reply:
On the principle I agree w/ @andrepeiry but I think that the way that the code is implemented here works just fine